guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -101,6 +101,18 @@ You must migrate a self-registered {Project} to the Red Hat Content Delivery Net
 To migrate a self-registered {Project} to the CDN, see {UpgradingDocURL}Upgrading_Server_upgrade-guide[Upgrading {ProjectName}] in the _Upgrading and Updating {ProjectName}_ guide.
 endif::[]
 
+ifdef::satellite[]
+.Considerations for Upgrades of {Project} to Future Versions
+
+When you upgrade to {Project} 6.10, you are required to upgrade to Pulp 3, which can be time consuming due to Pulp content migration.
```

Comment:
Welllll you are not *required* to migrate while on Satellite 6.9, it is *highly recommended*. If you perform no migrations on 6.9, they will just **all** be performed when you upgrade to 6.10. For smaller content setups, or those who can take longer downtimes this might be the chosen option.

Reply:
@ehelms I'm fuzzy on your last sentence. I'm assuming that the pre-upgrade Pulp content migration is recommended in order to reduce the down time during the actual upgrade to 6.10. So I would expect that for smaller content it's okay to proceed with automated migration during the upgrade instead of pre-update migration, which I would go with for larger volumes of content in Pulp.
What am I missing?

Reply:
Oh, by "those who can take longer downtimes" you mean those users who can afford longer down times. Now I probably get it :)